pkg/scheduler/plugins/elasticquota/elastic_quota_transformer.go

```diff
@@ -0,0 +1,75 @@
+/*
+Copyright 2022 The Koordinator Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package elasticquota
+
+import (
+	"k8s.io/client-go/tools/cache"
+	"k8s.io/klog/v2"
+	"sigs.k8s.io/scheduler-plugins/pkg/apis/scheduling/v1alpha1"
+	"sigs.k8s.io/scheduler-plugins/pkg/generated/informers/externalversions"
+
+	apiext "github.com/koordinator-sh/koordinator/apis/extension"
+	"github.com/koordinator-sh/koordinator/pkg/util/transformer"
+)
+
+func setupElasticQuotaTransformers(factory externalversions.SharedInformerFactory) {
+	resource := v1alpha1.SchemeGroupVersion.WithResource("elasticquotas")
+	informer, err := factory.ForResource(resource)
+	if err != nil {
+		klog.Fatalf("Failed to create informer for resource %v, err: %v", resource.String(), err)
+	}
+	if err = informer.Informer().SetTransform(TransformElasticQuota); err != nil {
+		klog.Fatalf("Failed to SetTransform in informer, resource: %v, err: %v", resource, err)
+	}
+}
+
+var elasticQuotaTransformers = []func(elasticQuota *v1alpha1.ElasticQuota){
+	TransformElasticQuotaWithDeprecatedDeviceResources,
```

Comment:
Are existing ElasticQuota using deprecated Device resources? If you don't need it, or remove these functions first, but keep the overall transformer mechanism.

Reply:
fixed